### ISSUE TITLE
 JD services only exists in the quickstart package

### ISSUE THREAD

**Issue by shoulders** [joomdev/jd_seattle#8]
This extension is installed in the quickstart package but does not seem to exist anywhere else. I suspect that this extension has been renamed or was a custom version of another extension.

![image](https://user-images.githubusercontent.com/319997/66050916-f34d7080-e525-11e9-89fd-89f11ab27f70.png)

**Comment by Nishtha003** [comment]
Yes, this extension is only specific to JD Seattle template and we'll add the placeholder there.

**Comment by shoulders** [comment]
can it have its own repo on github. would help prevent multiple versions of the module.

I think there is also a version in JD paris. both of which are slightly different
